benchmarks/viscoelastic_sheared_torsion/viscoelastic_sheared_torsion.prm

```diff
@@ -0,0 +1,169 @@
+# This parameter file constructs a 3D cube which is rotated around its vertical axis and sheared in the
+# xz-plane. It is adapted from Farrington et al. 2014 (Geophysics, Geochemsitry, Geosystems), section "3.4.
+# Torsion Test in 3-D". According to the authors this is essentially an extension of the simple shear test
+# (see the viscoelastic_plastic_simple_shear benchmark prm file) into the third dimension.
+# While the model always rotates with a constant rate of 42/360 per unit time, shearing is only applied until
+# t = 0.5. At this point the shearing velocity is reduced to zero, and elastic stresses relax.
+# For t < 0.5 the analytical expression of the stress is given by:
+#   tau = exp(-mu/eta*t)*(C2*cos(V*t/h)-C1*sin(V*t/h))-C2
+# and for t >= 0.5:
+#   tau = exp(-mu/eta*tmax)*(C2*cos(V*tmax/h)-C1*sin(V*tmax/h))-C2)*exp(-mu/eta*(t-tmax))
+# with:
+#   tmax = 0.5 [s]     # time at which shearing stops
+#   eta = 100  [Pa s]  # the shear viscosity
+#   mu  = 100  [Pa]    # the shear modulus
+#   V = 0.3    [m/s]   # the shearing velocity
+#   h = 1      [m]     # the height of the box
+#   omega = 42         # the dimensionless rotation rate
+#   C1 = -(V^2*eta^2*mu)/(mu^2*h^2+V^2*eta^2)
+#   C2 = -(V*h*eta*mu^2)/(mu^2*h^2+V^2*eta^2)
+#
+# These analytical stresses can be directly compared to the xz-component of the stress output from this parameter file.
+# Farrington et al., however, look at the stresses in an unrotated frame (their figure 1b), in which case both,
+# the analytical stress and the aspect stress component need to be multiplied with cos(omega*t). Note that in the
+# Farrington paper the vertical axis is the y-axis, such that their stress component xy is aspect's stress
+# component xz.
+# The fit of the numerical stress to the analytical solution can be enhances by choosing a smaller time step size.
+
+
+# Global parameters
+set Dimension                                  = 3
+set Start time                                 = 0
+set End time                                   = 2
+set Use years in output instead of seconds     = false
+set Resume computation                         = auto
+set CFL number                                 = 1.0
+set Maximum time step                          = 0.01
```

Comment:
Can you add a comment that due to this limitation, all timesteps are 0.01? (Right?)